src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/prometheus/silence-matcher-modal/silence-matcher-modal.component.html

```diff
@@ -1,6 +1,6 @@
 <cd-modal [modalRef]="bsModalRef">
-  <ng-container class="modal-title"
-                i18n>Matcher</ng-container>
+  <span class="modal-title"
+        i18n>{editMode, select, 1 {Edit} other {Add}} Matcher</span>
```

Comment:
In the other files you changed `1` to `true`, should we also do it in this file?
```suggestion
        i18n>{editMode, select, true {Edit} other {Add}} Matcher</span>
```

Reply:
To be consistent I changed it.